File: 2P/Atividade01.java
import java.util.Random;
import java.util.Scanner;
import java.util.ArrayList;

public class Atividade01 {
    public static void main(String[] args) {
        Random gerador = new Random();


        Scanner input = new Scanner(System.in);
        int tamanho = input.nextInt();

    }

    public void gerarLista(int tamanho, Random gerador){

        ArrayList<Integer> lista = new ArrayList<>();

        for (int i = 0; i < 100; i++){
            System.out.println(gerador.nextInt());
            System.out.println("aqui tem o imput");
        }

        for (int i = 0; i <= tamanho; i++){
            lista.add(gerador.nextInt());
        }

        for (int i : lista){
            if( i % 3 == 0){
                System.out.println(i + "esse numero é multiplo de 3");
            }
            else if(i % 2 == 0){
                System.out.println(i + "esse numero é par");
            }
            else {
                System.out.println(i + "esse numero é impar");
            }
        }
    }
}

File: 2P/Atividade01/Atividade01.py
import random

lista = []
tamanho = int(input("Tamanho: "))

def funcaoLista():

    for i in range(tamanho):
        lista.append(random.randint(1, 100))
    print(lista)
    
    for i in lista:
        if i % 3 == 0:
            print(i, "esse numero é modulo de 3")
        
        elif i % 2 == 0:
            print(i, "esse numero é par")
            
        else:
            print(f"{i} esse numero é ímpar")
    return "lista"

print(funcaoLista())